### DISCUSSION TITLE
How to run the simulation with 2 different laser wavelengths with slight delay

### DISCUSSION THREAD

**Discussion by vplj** [epochpic/epoch#235] [Q&A]
I would like to run the simulation with 2 different laser wavelengths with slight delay .
Do I have to use specify laser block for each wavelength? 
What else do I have to add in the input file.

**Comment by TomGoffrey** [comment]
@vplj I've converted the issue to a Q&A discussion.

If you want to have two lasers with different wavelengths, you will need two different laser blocks, as you say. I think this should be sufficient, but if not, let me know.
